db/migrate/20200910143249_fix_course_application_open_from_dates.rb

```diff
@@ -0,0 +1,14 @@
+class FixCourseApplicationOpenFromDates < ActiveRecord::Migration[6.0]
+  def up
+    recruitment_cycle = RecruitmentCycle.find_by(year: "2021")
+    return if recruitment_cycle.nil?
+
+    courses = recruitment_cycle.courses.where.not(applications_open_from: recruitment_cycle.application_start_date..recruitment_cycle.application_end_date)
```

Comment:
Can we split this line up a bit...

```suggestion
   valid_start_date_range = recruitment_cycle.application_start_date..recruitment_cycle.application_end_date
   courses = recruitment_cycle.courses.where.not(applications_open_from: valid_start_date_range)
```